packages/babel-traverse/src/path/family.js

```diff
@@ -209,7 +209,6 @@ export function getOuterBindingIdentifiers(duplicates?: boolean): Object {
   return t.getOuterBindingIdentifiers(this.node, duplicates);
 }
 
-// original source - https://github.com/babel/babel/blob/master/packages/babel-types/src/retrievers.js
```

Comment:
This link is 404

Reply:
https://github.com/babel/babel/tree/master/packages/babel-types/src/retrievers?